<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>893</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Индивидуальный план</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res.qrc">
    <normaloff>:/img/img/icon.png</normaloff>:/img/img/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="FacultyPanel" name="w_facultyPanel" native="true"/>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="Header" name="w_header" native="true"/>
        </item>
        <item>
         <widget class="QStackedWidget" name="stackedWidget">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="page">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>План не создан</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="btn_create">
                <property name="text">
                 <string>Создать</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>773</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_2">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QTabWidget" name="tabWidget">
              <property name="tabPosition">
               <enum>QTabWidget::North</enum>
              </property>
              <property name="tabShape">
               <enum>QTabWidget::Rounded</enum>
              </property>
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="FormTotalTime" name="tab_totalTime">
               <attribute name="title">
                <string>Сводка</string>
               </attribute>
              </widget>
              <widget class="EducationalWorkComplite" name="tab_educationFactical">
               <attribute name="title">
                <string>Выполнение учебной нагрузки</string>
               </attribute>
              </widget>
              <widget class="FormEducationWork" name="tab_educationWork">
               <attribute name="title">
                <string>I. Учебная работа</string>
               </attribute>
              </widget>
              <widget class="FormGenerikWork" name="tab_metod">
               <attribute name="title">
                <string>II. Учебно-методическая работа</string>
               </attribute>
              </widget>
              <widget class="FormGenerikWork" name="tab_research">
               <attribute name="title">
                <string>III. Научно-исследовательская работа</string>
               </attribute>
              </widget>
              <widget class="FormGenerikWork" name="tab_sport">
               <attribute name="title">
                <string>IV. Воспитательная и спортивная работа</string>
               </attribute>
              </widget>
              <widget class="FormGenerikWork" name="tab_other">
               <attribute name="title">
                <string>V. Другие виды работ</string>
               </attribute>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="a_exit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_print"/>
  </widget>
  <action name="a_exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_print">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/img/img/printer.png</normaloff>:/img/img/printer.png</iconset>
   </property>
   <property name="text">
    <string>Печать</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Header</class>
   <extends>QWidget</extends>
   <header>header/header.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FormTotalTime</class>
   <extends>QWidget</extends>
   <header>totaltime/formtotaltime.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FormEducationWork</class>
   <extends>QWidget</extends>
   <header>educationwork/formeducationwork.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FormGenerikWork</class>
   <extends>QWidget</extends>
   <header>generic/formgenerikwork.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EducationalWorkComplite</class>
   <extends>QWidget</extends>
   <header>educationwork/educationalworkcomplite.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FacultyPanel</class>
   <extends>QWidget</extends>
   <header>faculty/facultypanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
